client/knowledge_base_search/src/containers/QuestionBarContainer/styles.scss

```diff
@@ -8,6 +8,7 @@
 .question_bar {
   &--container {
     display: flex;
+    margin-left: -0.5rem;
```

Comment:
I think that you should fix the `&--button` margin instead for the `:first-child`